examples/docker-hello-world/docker-hello-world.bal

```diff
@@ -0,0 +1,10 @@
+import ballerina/http;
+
+// This code is completely focused on the business logic and it does not specify anything related to operations.
```

Comment:
This comment is not that meaningful, what is meant by operations here? @xlight05

Reply:
by operations, we are talking about the deployment here.
The idea was to refer to code as dev and deployment as operations(ops).

We can replace operations with deployment. seems like its more user-friendly.

Reply:
Changed to: `This code is completely focused on the business logic and it does not depend on the deployment.`